### ISSUE TITLE
Used memory is much bigger than configured capacity

### ISSUE THREAD

**Issue by kfchu** [snazy/ohc#68]
When I configured the max capacity to 10GB, the actual off-heap memory usage is 16GB, where does the extra 6GB come ?

Scenario:
- capacity: 10GB
- segment: 512
- hashtablesize: 32768
- linkedLRUMap
- key+value size: 1KB

**Comment by jgq2008303393** [comment]
Looking forward to more answers.

**Comment by snazy** [comment]
Yes, that's rather expected, because there's overhead with each allocation + linked-implementation. The chunked-implementation is more efficient for rather small entries, but expires/evicts only whole chunks.
